compendium/LogServiceImpl/test/TestLogService.cpp

```diff
@@ -60,6 +60,8 @@ class LogServiceImplTests : public ::testing::Test
         std::string text = oss.str();
         std::smatch m;
         bool found = std::regex_search(text, m, std::regex(regex));
+        std::cout << "TEXT STRING =" << text << std::endl;
+        std::cout << "REGEX = " << regex << std::endl;
```

Comment:
are these lines still needed? seems like these were for debugging and can be removed?